lib/main.js

```diff
@@ -196,12 +196,10 @@ export default {
         if (version.major > 2) {
           // PHPCS v3 and up support this with STDIN files
           parameters.push(`--ignore=${ignorePatterns.join(',')}`);
-        } else {
+
           // We must determine this ourself for lower versions
```

Comment:
This throws an error in the linter that it's an unneeded block inside that if. 
```js
} else {
  if (...)
}
```
I could take the `ignorePatterns.some` and set it to a variable then check that variable in an if block and that may work. 

```js
} else {
  const hasMatchedPattern = ignorePatterns.some(pattern => minimatch(filePath, pattern));
  if (hasMatchedPattern) {
    return [];
  }
}
```